.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,55 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+  push:
+    tags:
+      - 'v.[0-9]+.[0-100]+.[0-100]+'
+    branches:
+      - 'main'
+
+jobs:
+  snyk_scan_test:
+    if: ${{ github.event_name == 'pull_request' }}
+    strategy:
+      matrix:
+        depsfiles: [
+          requirements_docs.txt,
+          requirements_extra.txt,
+        ]
```

Comment:
Don't you want to add `requirements_tests.txt` here?

Reply:
Dependencies related to production were considered. Dev dependencies were ignored.